handlers/whatsapp_legacy/handler_test.go

```diff
@@ -570,58 +568,82 @@ var defaultSendTestCases = []OutgoingTestCase{
 		},
 		ExpectedMsgStatus: "W",
 		ExpectedExtIDs:    []string{"157b5e14568e8"},
-		SendPrep:          setSendURL,
 	},
 	{
-		Label:               "Unicode Send",
-		MsgText:             "☺",
-		MsgURN:              "whatsapp:250788123123",
-		MockResponseBody:    `{ "messages": [{"id": "157b5e14568e8"}] }`,
-		MockResponseStatus:  201,
-		ExpectedRequestBody: `{"to":"250788123123","type":"text","text":{"body":"☺"}}`,
-		ExpectedMsgStatus:   "W",
-		ExpectedExtIDs:      []string{"157b5e14568e8"},
-		SendPrep:            setSendURL,
+		Label:   "Unicode Send",
```

Comment:
I've been deleting these separate "Unicode Send" test cases... it only makes sense for mobile carriers who require different encodings for non-ASCII... they aren't needed for things like WhatsApp